react/package.json

```diff
@@ -3,10 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "bootstrap": "^4.0.0-beta.3",
```

Comment:
Nowadays we do not use `bootstrap` since it introduces style definitions for common DOM elements, like buttons, although I think this is not a problem for the training per se.
For future reading: https://speakerdeck.com/vjeux/react-css-in-js